cdk/names.py

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
so an idea i had for pre-injecting the stack name would be to use a module-level `__getattr__`, Just add to the top top of the module like so:

```
from .helpers import getenv

def __getattr__(attr):
  STACK_NAME =  getenv("STACK_NAME")
  if attr in globals():
    name = globals()[attr]
    return f"{STACK_NAME}-{name}"
  raise AttributeError(f"module {__name__} has no attribute {attr}") 
```